projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -136,6 +137,14 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     }
 
     this.setProps({ ...this.props, ...props });
+
+    if (this.checkContentChanges && isChanged<NgChanges<TippyDirective>>('content', changes)) {
+      if (!changes.content.previousValue && changes.content.currentValue) {
+        this.createInstance();
+      } else if (changes.content.previousValue && !changes.content.currentValue) {
+        this.destroyInstance();
```

Comment:
You also need to destroy the view ref.
There are a lot of cases handled in listeners within the ngAfterInit, You need to check that there are no collisions.

Reply:
destroyView added